plugin/src/main/java/com/denizenscript/denizen/objects/NPCTag.java

```diff
@@ -1277,6 +1277,98 @@ else if (attribute.startsWith("list", 2)) {
             return result;
         });
 
+        // <--[tag]
+        // @attribute <NPCTag.waypoint_provider>
+        // @returns ElementTag
+        // @description
+        // Returns the NPC's current Waypoint Provider type.
```

Comment:
Meta here should give some link and/or examples indicate what the valid provider types are